src/black/__init__.py

```diff
@@ -780,7 +777,7 @@ def get_sources(
                 continue
 
             if verbose:
-                out(f'Found input source: "{normalized_path}"', fg="blue")
```

Comment:
We don't use the value of normalized_path, we do `sources.add(path)`, so no point logging it